### ISSUE TITLE
Undefined index

### ISSUE THREAD

**Issue by jvoisin** [jbleuzen/Picturo#5]
There are some undefined session-related quirks. It seems that _$_SESSION['authed']_ is not set in the constructor of the class Picturo.

```
[Sun Sep 15 20:04:22 2013] PHP Notice:  Undefined index: authed in /home/jvoisin/dev/Picturo/core/picturo.php on line 218
[Sun Sep 15 20:04:22 2013] PHP Notice:  Undefined index: authed in /home/jvoisin/dev/Picturo/core/picturo.php on line 219
```

**Comment by jbleuzen** [comment]
Yeah I know there are some errors in logs.
I've been working to delete these errors on a private branch.

Thanks for the heads up !

**Comment by jbleuzen** [comment]
This issue should be fixed with release 0.2.0.
